src/components/GameCardsPage/GameCard.js

```diff
@@ -14,35 +14,38 @@ GameCard.propTypes = {
     id: PropTypes.string.isRequired,
     onDelete: PropTypes.func.isRequired,
     onEdit: PropTypes.func.isRequired,
+    showSaveGamePage: PropTypes.func.isRequired,
 }
 
-export default function GameCard({location, date, players, winner, shots, id, onDelete, onEdit}) {
+export default function GameCard({location, date, players, winner, shots, id, onDelete, onEdit, showSaveGamePage}) {
  
     const [isSetToDelete, setIsSetToDelete] = useState(false)
 
     return(
         <Card>
             {isSetToDelete || (
-                <SavedGameContent>
-                    <Location>{location}</Location>
-                    <Date>{date}</Date>
-                    <PlayerWrapper>
-                        <h4>Player(s)</h4>
-                        <span>{players}</span>
-                    </PlayerWrapper>
-                    <WinnerWrapper>
-                        <h4>Winner(s)</h4>
-                        <span>{winner}</span>
-                    </WinnerWrapper>
-                    <ShotsWrapper>
-                        <h4>Total Shots</h4>
-                        <span>{shots}</span>
-                    </ShotsWrapper>
+               <> 
+                    <SavedGameContent>
+                        <Location>{location}</Location>
+                        <Date>{date}</Date>
+                        <PlayerWrapper>
+                            <h4>Player(s)</h4>
+                            <span>{players}</span>
+                        </PlayerWrapper>
+                        <WinnerWrapper>
+                            <h4>Winner(s)</h4>
+                            <span>{winner}</span>
+                        </WinnerWrapper>
+                        <ShotsWrapper>
+                            <h4>Total Shots</h4>
+                            <span>{shots}</span>
+                        </ShotsWrapper>
+                    </SavedGameContent>
                     <ButtonWrapper>
-                        <ButtonDeleteIcon onClick={() => setIsSetToDelete(true)} data-testid="button-delete-icon"><BinIcon/></ButtonDeleteIcon>
-                        <ButtonEditIcon onClick={() => onEdit(id)} data-testid="button-edit-icon"><PenIcon/></ButtonEditIcon>
+                            <ButtonDeleteIcon onClick={() => setIsSetToDelete(true)} data-testid="button-delete-icon"><BinIcon/></ButtonDeleteIcon>
+                            <ButtonEditIcon onClick={() => handleEdit(id)} data-testid="button-edit-icon"><PenIcon/></ButtonEditIcon>
```

Comment:
I would name the two data-testids `button-set-delete` and `button-edit`. Or something similar, but without the ending -icon.